conans/test/integration/toolchains/cmake/test_cmake.py

```diff
@@ -31,3 +31,52 @@ def run(self, *args, **kwargs):
     assert "-something" in client.out
     assert "--testverbose" in client.out
     assert "-testok" in client.out
+
+def test_build_type_single_config_warn():
```

Comment:
These tests passes in PRs because they run only in Linux in PRs. After merge, they will run in Windows too and fail (these tests fail in Windows due to multi-config VS). It would be better to move them to "functional" folder, so they run in all platforms also in PRs.

Reply:
I think the best is to do actual builds, with fully functional examples in the "functional" folder, building libraries with full ``test_package`` functionality. This will make sure things are working.

Reply:
I have created some tests for my all my use cases. I always set build_type via `-s build_type=Debug`